indexer/src/store.rs

```diff
@@ -286,10 +286,12 @@ impl<CS: ChainStore + 'static> DefaultIndexerStore<CS> {
             .expect("none empty index states")
             .block_number;
 
-        // need to lock chain state, avoids inconsistent state in processing
-        let chain_state = self.shared.lock_chain_state();
+        let (tip_number, tip_hash) = {
+            let chain_state = self.shared.lock_chain_state();
```

Comment:
Consider using `store.get_tip_header` to completely avoid the lock?

Reply:
> Consider using `store.get_tip_header` to completely avoid the lock?

Resolved, please review again, thanks.